maintainers/README.md

```diff
@@ -0,0 +1,179 @@
+<!--
+This document was originally based on the process used by Open vSwitch (OVS).
+You can find information about the OVS committers and the related processes
+used by OVS here: http://docs.openvswitch.org/en/latest/internals/maintainers/
+-->
+
+# Maintainers
+
+A Metal3 maintainer is a participant in the project with the ability to approve
+changes proposed to a given repository. Approval access grants a broad ability
+to affect the progress of the project as presented by its most important
+artifact, the code and related resources. As such it represents a significant
+level of trust in an individual's commitment to working with other maintainers
+and the community at large for the benefit of the project. It can not be
+granted lightly and, in the worst case, must be revocable if the trust placed
+in an individual was inappropriate.
+
+This document governs how maintainers are added or removed from each
+of the Metal3 project repositories.  The current list of maintainers of a given
+repo can be found in the `OWNERS` file of each repository.
+
+This document suggests guidelines for granting and revoking approval access. It
+is intended to provide a framework for evaluation of such decisions without
+specifying deterministic rules that wouldn't be sensitive to the nuance of
+specific situations. In the end the decision to grant or revoke maintainer
+privileges is a judgment call made by the existing set of maintainers.
+
+## Granting Approval Access
+
+Granting approval access should be considered when a candidate has demonstrated
+the following in their interaction with the project:
+
+- Contribution of significant new features through the patch submission
+  process where:
+
+  - Submissions are free of obvious critical defects
+  - Submissions do not typically require many iterations of improvement
+    to be accepted
+
+- Consistent participation in code review of other's patches, including
+  existing maintainers, with comments consistent with the overall project
+  standards
+
+- Assistance to those in the community who are less knowledgeable through
+  active participation in project forums such as github, slack, or the
+  metal3-dev mailing list.
+
+- Plans for sustained contribution to the project compatible with the project's
+  direction as viewed by current maintainers
+
+The process to grant approval access to a candidate is as follows:
+
+- An existing maintainer nominates the candidate by sending an email to all
+  existing maintainers with information substantiating the contributions of the
+  candidate in the areas described above.
```

Comment:
Is this meant to be a private email thread, or public on the mailing list?

Reply:
Good point - private.  I will update that to clarify.